test/helpers/cilium.go

```diff
@@ -97,37 +97,37 @@ func (s *SSHMeta) GetEndpointMutableConfigurationOption(endpointID, optionName s
 }
 
 // SetAndWaitForEndpointConfiguration waits for the endpoint configuration to become a certain value
-func (s *SSHMeta) SetAndWaitForEndpointConfiguration(endpointID, optionName, expectedValue string) bool {
-	maxRetries := 10
-
-	log.Infof("Setting endpoint %s config %s=%s", endpointID, optionName, expectedValue)
-
-	// HACK:
-	// Retry multiple times as policy generations can still be queued up in
-	// which case the value will immediately swap back
-	for retry := 1; retry <= maxRetries; retry++ {
+func (s *SSHMeta) SetAndWaitForEndpointConfiguration(endpointID, optionName, expectedValue string) error {
+	logger := s.logger.WithFields(logrus.Fields{
+		"ep":     endpointID,
```

Comment:
We have `pkg/logfields` which has a variety of constants like `logfields.EndpointID`. Having uniformity across CI and Cilium with these fields makes it easier for developers to debug since it is similar to what Cilium has. Can you use `logfields.EndpointID`?